lib/fog/openstack/requests/identity_v3/list_role_assignments.rb

```diff
@@ -3,33 +3,39 @@ module Identity
     class OpenStack
       class V3
         class Real
-          def list_role_assignments(options={})
-            params = Hash.new
-            params['group.id'] = options[:group_id] if options[:group_id]
-            params['role.id'] = options[:role_id] if options[:role_id]
-            params['scope.domain.id'] = options[:domain_id] if options[:domain_id]
-            params['scope.project.id'] = options[:project_id] if options[:project_id]
-            params['user.id'] = options[:user_id] if options[:user_id]
-            params['effective'] = options[:effective] if options[:effective]
-
-            params['page'] = options.fetch(:page, 1)
-            params['per_page'] = options.fetch(:per_page, 30)
+          def list_role_assignments(options = {})
+            # Backwards compatibility name mapping
+            name_mapping = {
```

Comment:
The reason for this is that in this API call (according to http://developer.openstack.org/api-ref-identity-v3.html#listRoleAssignments) the parameter names have dots in them, which is not particularly Ruby-friendly. In the rest of the Identity V3 API the parameter names usually have underscores instead of dots.